.github/scripts/rust/setup.py

```diff
@@ -101,40 +96,55 @@ def filter_for_publishable_crates(crates):
     return [crate for crate in crates for pattern in PUBLISH_PATTERNS if fnmatch(crate, pattern)]
 
 
-def output_exclude(crates):
+def output_matrix(name, crates, **kwargs):
     """
-    Prints a exclude statements for a GitHub Action matrix.
-
-    Currently, this only excludes nightly-only crates from running on stable by default
+    Outputs the job matrix for the given crates
+    :param name: The name where the list of crates will be stored to be read by GitHub Actions
     :param crates: a list of paths to crates
     """
 
-    output = json.dumps([dict(toolchain="stable", directory=str(crate)) for crate in filter_for_nightly_only_crates(crates)])
-    print(f"::set-output name=exclude::{output}")
-    print(f"exclude = {output}")
+    available_toolchains = []
+    for crate in crates:
+        for pattern, additional_toolchains in TOOLCHAINS.items():
+            toolchain_toml = open(crate/"rust-toolchain.toml", "r")
```

Comment:
Curious about this syntax. Does pathlib overload the `/` operator to be path concat?

Reply:
Yes 🙂

Reply:
It does and I love it. I think generally people put a space around the operator though to be clearer

Reply:
Is this a change request? ^ 🙃

Reply:
I was going to do a review tomorrow morning, but yes I think I am asking for a space 😁 

We should consider running pylint on this locally or something

Reply:
Applied pylons and reformatted the code: fa2964e36acc8b7957ec9c005238b05de96a20d8

Reply:
Thanks